### PR TITLE
Update default Console version to v2.3.2

### PR DESCRIPTION
**Pull request by weeco**
Maybe we can automate this, since this is going to be a common task.

### CODE REVIEW

File: charts/console/Chart.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ type: application
 version: 0.7.0
```
Comment (alejandroEsc) [inline charts/console/Chart.yaml:30]: Could you please bump the version also. 
```suggestion
version: 0.7.1
```

**Review by alejandroEsc — COMMENTED** [review]
Just one ask.

**Comment by RafalKorepta** [comment]
NIT: We have automation already in place
https://github.com/redpanda-data/helm-charts/blob/6a59cbd1c42dc465868900d960025f2d4648340e/.github/workflows/nightly_version_checks.yaml#L28

I wonder why it didn't pick latest console.

**Comment by RafalKorepta** [comment]
The difference is how console represent container registry with container image.

https://github.com/redpanda-data/helm-charts/blob/6a59cbd1c42dc465868900d960025f2d4648340e/charts/console/values.yaml#L22-L26

vs

https://github.com/redpanda-data/helm-charts/blob/6a59cbd1c42dc465868900d960025f2d4648340e/charts/redpanda/values.yaml#L45-L48

Maybe [`docker-tag-list`](https://github.com/joejulian/docker-tag-list) doesn't default to `docker.redpanda.com` registry and can not find [`v2.3.2`](https://hub.docker.com/layers/redpandadata/console/v2.3.2/images/sha256-07430db46a063a75ea506ddff1a0533d6d1a8c93628aaf4b207b0d03128014ad?context=explore) tag

The output of the github action job is not helpful https://github.com/redpanda-data/helm-charts/actions/runs/6091589858/job/16528414465#step:6:5

**Review by RafalKorepta — APPROVED** [review]

**Pull request merged**